gem/tests/test_auth.py

```diff
@@ -118,6 +118,25 @@ def test_add_user_role_from_claims(self):
         self.assertEquals(user.groups.all().count(), 1)
         self.assertEquals(user.groups.first().name, 'data_admin')
 
+    def test_display_username_as_alias_from_claims(self):
```

Comment:
Just a tip for the test name: make sure it's accurate for the test. In this case you're not testing the display or updating the alias from the claims. You're testing that the alias is always set to be the same as the username.